### DISCUSSION TITLE
Welcome to Browser Extension Wallet Discussions!

### DISCUSSION THREAD

**Discussion by UrAvgDeveloper** [fetchai/asi-alliance-wallet#39] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TooBroke4U** [comment]
New user, trying to get a better view of what is going on before I commit fully and understanding of it all.

**Comment by m-hesam-moradian** [comment]
Hi! I’m really excited to get involved in this community. I’m looking for a team where we can sit down together, solve Issues, and develop collaboratively.
